src/libs/utils.js

```diff
@@ -231,6 +232,34 @@ function getHdPath(address) {
   return stringToPath(hdPath)
 }
 
+function isEvmosBasedChain(chainId) {
+  const re = /[_]{1}[\d]{4}[\\-]{1}[\d]+$/g
```

Comment:
used this regex to work with point network and evmos chainid
 `re = /[_]{1}[\d]{4,5}[\\-]{1}[\d]+$/g`